document_page_work_instruction/views/document_page_work_instructions.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record model="ir.actions.act_window" id="work_instructions_action">
+      <field name="name">Work Instructions</field>
+      <field name="res_model">document.page</field>
+      <field name="view_type">form</field>
+      <field name="view_mode">tree,form</field>
+      <field name="help">Work instructions of your management system.</field>
+      <field name="domain" eval="[('parent_id','=',ref('document_page_group_work_instructions'))]"/>
```

Comment:
I don't know your opinion, but I think that managing the category of the work instruction through the parent is something fishy... 
I think it would be better to manage it through an inherits or a boolean invisible field, I don't know what do you think, but this kind of management could give problems (users may change the category and the work instructions disappear...).
The change should be managed on a migration script.

Reply:
Although I don't have a formed opinion on this, it's been like this since previous versions. Maybe this should be a separate PR as an IMP.

Reply:
I think that prevent the user to change the category should be a good way to fix this problem.

Reply:
Is this going to be attended in this PR or on a new one?

Reply:
Follow up here: https://github.com/OCA/management-system/issues/237